stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -134,6 +135,17 @@ function define_default_editors()
                 `$cmd "+normal $(line)G$(column)|" $path`
         end
     end
+    # nvr (neovim-remote)
+    define_editor("nvr"; wait=true) do cmd, path, line, column
+        # This will not block (even with wait=true) if there is a running
+        # server process since then nvr just fires the command and exits.
+        # wait=true is still needed in the case where nvr will start the neovim
+        # process.
+        cmd = `$cmd --remote-silent`
```

Comment:
Just FYI, if you use `$cmd --remote-wait-silent` it will block. Is there a reason you didn't use that? Your comment makes it sound like you want it to block but `nvr` doesn't support that feature.

btw, when using `nvr` with a terminal inside Neovim, I prefer the non-blocking behavior. If you use `--remote-wait`, `nvr` will only unblock and return when the buffer it opens is closed, which can be hard to track down if you switch between multiple files while debugging, and could be non-intuitive.

So I think what you have here, i.e. `--remote-silent` is great. I'd would probably add `wait=false` in the `define_editor` function:

```
define_editor("nvr"; wait=false) do cmd, path, line, column
```

and change the comment to say that we choose to not block on purpose.

Reply:
My comment was unclear then. It behaves as I want it to: if there is a running process it should _not_ block, but if there isn't, then we _have_ to block (so can't have `wait=false`).

However, I realized just now that this doesn't work with `Alt+e` option to open the current REPL input in an editor, for this case we must use `--remote-wait-silent`, but I am not sure the current infrastructure allow different settings in different contexts like that.